src/Settings_Helper.php

```diff
@@ -4,7 +4,7 @@
 
 use Tribe__Main;
 
-if ( ! class_exists( 'Settings_Helper' ) ) {
+if ( ! class_exists( NS . 'Settings_Helper' ) ) {
```

Comment:
:thinking: I'm not opposed to keeping these, but in a world where we encourage and give an example of autoloading, they are pretty unnecessary.

Reply:
gotcha